src/wazuh_modules/vulnerability_scanner/testtool/rocksDBQuery/main.cpp

```diff
@@ -21,7 +21,7 @@ int main(const int argc, const char** argv)
     {
         CmdLineArgs cmdLineArgs(argc, argv);
 
-        auto rocksDB = Utils::RocksDBWrapper(cmdLineArgs.getDBPath());
+        auto rocksDB = Utils::RocksDBWrapper(cmdLineArgs.getDBPath(), cmdLineArgs.getColumnFamilies());
```

Comment:
I'm getting two duplicate outputs, is that correct? again, I don't think this is a stopper for this implementation.

Reply:
Too late to see the difference! Thanks!